transcode/transcode_test.go

```diff
@@ -76,11 +76,11 @@ func TestItCanTranscode(t *testing.T) {
 		tr: clients.TranscodeResult{
 			Renditions: []*clients.RenditionSegment{
 				{
-					Name:      "lowlowlow",
+					Name:      "360p",
 					MediaData: []byte("pretend media data"),
 				},
 				{
-					Name:      "super-high-def",
+					Name:      "720p",
```

Comment:
Just a note for real broadcaster: name must start with letter otherwise transcode fails

Reply:
I didn't realise, good to know thanks!